modules/ROOT/pages/policy-template-source.adoc

```diff
@@ -0,0 +1,36 @@
+= Policy's Template Source URL
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: policy, CloudFront
+
+[NOTE]
+--
+This article presents configurations parameters thar are only relevant for Mule versions in the ranges [4.1.7, 4.2.0) and [4.2.2,4.3.0).
```

Comment:
Since v4.3.0 is not released yet, do you know for sure that it won't be included there?

Reply:
Yes, the feature will die in runtime 4.3.0, not a single doubt about it